### ISSUE TITLE
Need way to pass cloud-init via stdin

### ISSUE THREAD

**Issue by gerboland** [canonical/multipass#617]
I need to allow reading cloud-init from stdin. So user can do something like:
```
$ multipass launch --cloud-init -<return>
#cloud-config
password: passw0rd
chpasswd: { expire: False }
<Ctrl+D>
```
and it will use that text as the cloud-init configuration

**Comment by Saviq** [comment]
#628 is up.

**Comment by townsend2010** [comment]
Closed via #628
